lib/gpu_stats.py

```diff
@@ -58,7 +58,9 @@ def initialize(self):
                     if self.logger:
                         self.logger.debug("OS is not macOS. Using pynvml")
                     pynvml.nvmlInit()
-                except pynvml.NVMLError_LibraryNotFound:
+                except (pynvml.NVMLError_LibraryNotFound, pynvml.NVMLError_DriverNotLoaded):
+                    if self.logger:
```

Comment:
This needs to go to line 34 as otherwise macOS won't pick up the warning

Reply:
Right, makes sense. Changed